patchwork-fml/src/main/java/cpw/mods/modlauncher/api/INameMappingService.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2019, 2019
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package cpw.mods.modlauncher.api;
```

Comment:
Could this cause issues if Fabric is launched using ModLauncher? I would assume the true class would get loaded in that case, however.

Reply:
Based on a bit of googling--as long as ModLauncher is loaded before Patchwork (which is true) it *should* just not load our stripped class.
The only way to know for absolutely sure is to test, but it should work fine theoretically.